ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -369,6 +369,37 @@ def test_torch_nanmean(
     )
 
 
+@handle_frontend_test(
+    fn_tree="torch.nanquantile",
+    dtype_and_x=_statistical_dtype_values(
+        function="nanquantile",
+    ),
+    keepdims=st.booleans(),
+)
+def test_torch_nanquantile(
+    *,
+    dtype_and_x,
+    keepdims,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+    backend_fw,
+):
+    input_dtype, x, axis = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x[0],

```

Comment:
you're also supposed to generate and pass the other arguments here as well.
`q`, `interpolation` etc.
All argument should be covered